packages/destination-actions/src/destinations/close/createUpdateContactAndLead/index.ts

```diff
@@ -0,0 +1,92 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Create Update Contact and Lead',
+  description: '',
+  defaultSubscription: 'type = "identify"',
+  fields: {
+    lead_name: {
+      label: 'Lead Name',
+      description: '',
+      type: 'string',
+      required: false,
+      default: {
+        '@path': '$.traits.company.name'
+      }
+    },
+    contact_name: {
+      label: 'Contact Name',
+      description: '',
+      type: 'string',
+      required: false,
+      default: {
+        '@path': '$.traits.name'
+      }
+    },
+    contact_email: {
+      label: 'Contact Email',
+      description: '',
+      type: 'string',
+      required: false,
+      default: {
+        '@path': '$.traits.email'
+      }
+    },
+    contact_phone: {
+      label: 'Contact Phone',
+      description: '',
+      type: 'string',
+      required: false,
+      default: {
+        '@path': '$.traits.phone'
+      }
+    },
+    contact_url: {
+      label: 'Contact Url',
+      description: '',
+      type: 'string',
+      required: false,
+      default: {
+        '@path': '$.traits.website'
+      }
+    },
+    contact_title: {
+      label: 'Contact Title',
+      description: '',
+      type: 'string',
+      required: false,
+      default: {
+        '@path': '$.traits.title'
+      }
+    },
+    contact_external_id: {
+      label: 'Contact External Id',
```

Comment:
Is there a better label than "external"? It's "external" for us at Close but not for Segment users.

"Custom User Identifier"?

Reply:
Let's capitalize "ID" here and everywhere user-facing.

I agree that "Contact External ID" has never felt super clear to me. I'm not sure what would be better though.

Remind me what the user is choosing here. In most cases they'll choose the Segment `User ID` field right? What are the other options?

Reply:
Perhaps "Contact Identifier", but I'm still unclear what the options are besides choosing userID

Reply:
@philfreo Yes, in most cases User ID is set here - the id how users are identified in the customer's system. For us it's the `user_xxxxx` format.
I would keep the word "Contact" in the field name since it is setting a field on a Contact. "Contact Identifier" is fine, although should not be confused with Close Contact ID (`cont_xxxx`).

Reply:
Changing to "Contact Identifier". Since it won't go public yet we can update or add more description later.